spaces/S000198/properties/P000017.md

```diff
@@ -0,0 +1,8 @@
+---
+space: S000198
+property: P000017 #sigma-compact
+value: true
+---
+
+This space is a countable union of {P17} spaces
```

Comment:
"This space is the union of ..." is simpler and good enough.

Reply:
```suggestion
This space is the union of {P17} spaces
```